sql/item_latents.sql

```diff
@@ -2777,22 +2776,298 @@ INSERT INTO `item_latents` VALUES(14646, 29, 10, 32, 0); -- Darksday: MDB+10
 -- -------------------------------------------------------
 -- LAVALIER
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES (10961,60,-40,13,14); -- CHARMED:LIGHTRES
-INSERT INTO `item_latents` VALUES (10961,61,-40,13,14); -- CHARMED:DARKRES
+INSERT INTO `item_latents` VALUES(10961,60,-40,13,14); -- CHARMED:LIGHTRES
```

Comment:
I've been actively trying to get our sql files to consistently be the standard output instead of whatever random thing ppl did or poorly configured tool they used (navicat should die in a fire using all those single quotes as kindling!)